fireworks/fw_config.py

```diff
@@ -17,7 +17,7 @@
 NEGATIVE_FWID_CTR = 0
 
 # this is where load_object() looks for serialized objects
-USER_PACKAGES = ["fireworks.user_objects", "fireworks.utilities.tests", "fw_tutorials", "fireworks.features"]
+USER_PACKAGES = ["fireworks.user_objects", "fw_tutorials", "fireworks.features"]
```

Comment:
@computron Is this change ok to do? I'm not sure of its implications.

My reason for doing this is:

I added some new tests that use `pytest` rather than `unittest` which now causes errors:

```py
lpad found /Users/janosh/Repos/dielectric-frontier/FW_config.yaml
/rds/user/jr769/hpc-work/.venv/py39/lib/python3.9/site-packages/fireworks/utilities/fw_serializers.py:361: UserWarning: None in fireworks.utilities.tests.test_visualize cannot be loaded because of No module named 'pytest'. Skipping..
  warnings.warn(f"{m_object} in {mod_name} cannot be loaded because of {str(ex)}. Skipping..")
Traceback (most recent call last):
  File "/rds/user/jr769/hpc-work/.venv/py39/lib/python3.9/site-packages/fireworks/utilities/fw_serializers.py", line 354, in load_object
    m_module = importlib.import_module(mod_name)
  File "/rds/user/jr769/hpc-work/.venv/py39/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/rds/user/jr769/hpc-work/.venv/py39/lib/python3.9/site-packages/fireworks/utilities/tests/test_visualize.py", line 1, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
```

The errors are triggered by this `load_function()`

https://github.com/materialsproject/fireworks/blob/f9f3fe73d2de6ee724ca6e14832ab6b2b04a66ca/fireworks/utilities/fw_serializers.py#L350-L362

Trying to import pretty much every package imported anywhere in Fireworks, even in test files.

Why is that necessary to load a simple YAML file? This seems like a bad idea in general. Second, why include test files here at all? Wouldn't the regular package files be enough to check for imports?